spec/rubocop/cop/layout/rescue_ensure_alignment_spec.rb

```diff
@@ -36,6 +36,30 @@
         end
       RUBY
     end
+
+    context 'as RHS of assignment' do
+      it 'accepts multi-line assignment' do
+        expect_no_offenses(<<-RUBY.strip_indent)
```

Comment:
I think we also need a similar example when the `begin` is on the same line as the assignment op.